.jshintrc

```diff
@@ -6,7 +6,10 @@
   "globals": {
     "define": true,
     "require": true,
-    "console": true
+    "console": true,
```

Comment:
Could we scope this config to the server directory? See https://github.com/elasticsearch/kibana/blob/master/tasks/.jshintrc